defaults/main.yml

```
@@ -0,0 +1 @@
+clamav_cron_frequency: "weekly"
```

We typically prefer to leave the quotes off unless they are necessary.
```suggestion
clamav_cron_frequency: weekly
```